velox/functions/prestosql/types/TimestampWithTimeZoneType.cpp

```diff
@@ -32,16 +32,14 @@ void castFromTimestamp(
   }
   const auto adjustTimestampToTimezone = config.adjustTimestampToTimezone();
 
-  auto timestampVector = rowResult.childAt(0)->asFlatVector<int64_t>();
-  auto timezoneVector = rowResult.childAt(1)->asFlatVector<int16_t>();
-  auto rawTsValues = timestampVector->values()->asMutable<int64_t>();
-  auto rawTzValues = timezoneVector->values()->asMutable<int16_t>();
-  timestampVector->clearNulls(rows);
-  timezoneVector->clearNulls(rows);
+  auto timestampWithTimezoneVector = result->asFlatVector<int64_t>();
```

Comment:
maybe use something like tsWithTz abbreviation for both the vector and raw buffer for consistency.

Reply:
Fixed, thanks.